src/tools/merssh/qmakecommand.cpp

```diff
@@ -65,15 +65,44 @@ int QMakeCommand::execute()
         return 1;
     }
 
+    QString evalCommand;
+    QString filterCommand; 
+    const int evalIndex = arguments().indexOf(QLatin1String("-env"));
```

Comment:
I would prefer more strict checking for this argument, like checking just the very first one (I do not remember if there are other arguments)